core/rust/src/ordered_hash_map.rs

```diff
@@ -5,6 +5,13 @@ pub struct OrderedHashMap<K, V>(linked_hash_map::LinkedHashMap<K, V>)
 where
     K: Hash + Eq + Ord;
 
+impl<K, V> Default for OrderedHashMap<K, V>
```

Comment:
I didn't derive `Default` as `ordered_hash_map::OrderedHashMap` only implements it when your generic parameters implement certain traits. We don't use a non-default hasher though so `new()` is fine for us.